src/Uno.UI.Composition/Composition/InsetClip.skia.cs

```diff
@@ -0,0 +1,17 @@
+#nullable enable
+using System;
+using System.Linq;
+using SkiaSharp;
+
+namespace Windows.UI.Composition;
+
+partial class InsetClip
+{
+	internal SKRect SKRect => new()
+	{
+		Top = TopInset - 1,
+		Bottom = BottomInset + 1,
+		Left = LeftInset - 1,
+		Right = RightInset + 1
```

Comment:
Those `1` and `-1` are very strange.

Reply:
Indeed ... this is just an extract of what was previously done directly in the `Compositor`, so I didn't review it and will not in the current PR.